docker/config_template.yaml

```diff
@@ -37,6 +38,7 @@ persistence:
                 user: {{ default .Env.CASSANDRA_USER "" }}
                 password: {{ default .Env.CASSANDRA_PASSWORD "" }}
                 port: {{ default .Env.CASSANDRA_PORT "9042" }}
+                maxConns: {{ default .Env.CASSANDRA_MAX_CONNS "10" }}
```

Comment:
This is probably too big. I think we can leave this unset or set it to 2? What do you think

Reply:
i guess 2 is too small